v2_migrations/2024-08-28-081838_update_v2_primary_key_constraints/down.sql

```diff
@@ -86,8 +86,17 @@ ALTER TABLE payment_intent
 ALTER COLUMN currency DROP NOT NULL,
     ALTER COLUMN client_secret DROP NOT NULL,
     ALTER COLUMN profile_id DROP NOT NULL;
-ALTER TABLE payment_intent ALTER COLUMN active_attempt_id SET NOT NULL;
-ALTER TABLE payment_intent ALTER COLUMN session_expiry DROP NOT NULL;
+
+ALTER TABLE payment_intent
+ALTER COLUMN active_attempt_id
+SET NOT NULL;
+
+ALTER TABLE payment_intent
+ALTER COLUMN session_expiry DROP NOT NULL;
+
+ALTER TABLE payment_intent
```

Comment:
We can't set this kind of default to active_attempt_id, this should be NULL.

Reply:
This is in down.sql, so this was how the previous application behaved